db/migrate/20210129122017_remove_null_constraints_from_application_experiences.rb

```diff
@@ -0,0 +1,6 @@
+class RemoveNullConstraintsFromApplicationExperiences < ActiveRecord::Migration[6.0]
+  def change
+    change_column :application_experiences, :details, :text, null: true
```

Comment:
`change_column_null` here could be preferable because you can't accidentally change a different attribute (like type). And I think it's reversible as well?

https://edgeapi.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_column_null

Reply:
Yup - I've made the change